tests/io/test_text.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2021 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Test text handling functions."""
+from datetime import datetime
+
+import numpy as np
+import shapely.geometry as sgeom
+
+from metpy.cbook import get_test_data
+from metpy.io import parse_wpc_surface_bulletin
+
+
+def test_parse_wpc_surface_bulletin():
```

Comment:
Are there already some reasonable cases out there to test more broadly against? Should we capture some of the cases documented in the code (eg multiple pressure feature locations per strength, strengthening qualifier strings for fronts)?

Reply:
The example dataset definitely has the "multiple centers with a single strength" issue. That thing broke the parser changes initially and I had to verify in the IEM archive that what we had matched the original product; the spec indicates that these should always be paired up, but here we are. 🤷‍♂️ 

OH. Apparently the basic "coded surface bulletin" has the wk/moderate/strong modifier, but not the "high res" bulletin. Because of course.

I'll beef up the testing with one of each bulletin.